sdk/formrecognizer/Azure.AI.FormRecognizer/tests/DocumentAnalysisClient/DocumentAnalysisClientLiveTests.cs

```diff
@@ -227,7 +227,7 @@ public async Task AnalyzeDocumentWithCustomModel(bool useStream)
             var modelId = Recording.GenerateId();
             AnalyzeDocumentOperation operation;
 
-            await using var customModel = await CreateDisposableBuildModelAsync(modelId);
+            await using var customModel = await BuildDisposableDocumentModelAsync(modelId);
```

Comment:
Renamed our test helper class `DisposableBuildModel` to `DisposableDocumentModel` to align with the terminology used by the service. This also aligns with our other helper class, `DisposableDocumentClassifier`.